packages/frontend/config/environment.js

```diff
@@ -47,6 +47,10 @@ module.exports = function (environment) {
         visualNoiseLevel: 1,
       },
     },
+    'ember-local-storage': {
```

Comment:
Wondering if we should turn off `includeEmberDataSupport` explicitly as well. I'm not exactly sure what it does, but don't want to cache stuff accidentally.

Reply:
good catch, i thought it was opt-in, but it's opt-out. 

done. :heavy_check_mark: